js/_actions.js

```diff
@@ -54,9 +58,9 @@ export function failRevisionsGet(err) {
 }
 
 export function getRevisions() {
-	return (dispatch) => {
+	return (dispatch, getState) => {
 		dispatch(startRevisionGet());
-		return api.getRevisions()
+		return api.call(dispatch, getState, '/gitrefs', 'get')
 			.then(json => dispatch(succeedRevisionsGet(json)))
```

Comment:
The action themselves know which dispatcher they are connected to, so they should now which url they would like to call.